recipes/librtlsdr/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+find_package(librtlsdr REQUIRED)
+
+add_executable(${PROJECT_NAME} test_package.cpp)
```

Comment:
It looks like librtlsdr is c-only.
It would be nice if you converted `test_package.cpp` to `test_package.c`.